app/components/nav-bar.js

```diff
@@ -8,5 +8,21 @@ export default Component.extend({
       this.get('authManager').logout();
       this.get('routing').transitionTo('index');
     }
+  },
+
+  didInsertElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
+      popup : '.popup',
+      on    : 'click'
+    });
+  },
+
+  willDestroyElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
```

Comment:
@niranjan94 
>Title black & bold (it's already bold)

> Description same color as time & bold
>Why the custom color and weight? The moment we start using custom colors like this for event basic elements we loose global theming capability.

@niranjan94 rgba(0, 0, 0, 0.87) I had tried to use before, wasn't a custom colour, but the same one used across the site.
Since it leads to loss of global theming, is it okay if I declare the colour as a separate class under `text.scss` as done for `.gray-text` class.